public/js/LoginAndSignup.js

```diff
@@ -89,9 +89,9 @@ function RegisterSuccessAlert (data)
 /*
 | RegistrErrorAlert
 | gives error report to user
-| receives 1 parameter 
+| receives 1 parameter
 */
-function RegistrErrorAlert () 
+function RegistrErrorAlert ()
```

Comment:
The function name should start with a small letter, and the spelling of this function name is wrong